.github/workflows/pullrequest-trusted.yml

```diff
@@ -16,7 +16,7 @@ on:
 jobs:
   test:
     if: (github.event.label.name == 'safe to test' || github.event.label.name == 'appimage')
-    uses: viamrobotics/rdk/.github/workflows/test.yml@main
+    uses: ./.github/workflows/test.yml
```

Comment:
This is just temporary, right?

Reply:
Yes, I was checking that the tests would pass on my branch first, this blocks it from getting merged anyway, right?